.github/workflows/test.yaml

```diff
@@ -55,6 +60,12 @@ jobs:
         python --version
         tox -q
 
+    - name: upload dist
```

Comment:
Can we skip this upload + download part and instead create the sdist + wheels in the pypi-publish job?

I don't see what is the difference between creating the wheels via tox or ciwheelbuilder.

My expectation is that we will get the exactly same wheel... and since we use tox-wheel to run the tests we will be testing the same wheel.

Reply:
> ciwheelbuilder

you mean cibuildwheel? that's for setting up specific docker containers with specific libraries for manylinux1 for statically linking against. We're just using plain old `pip wheel`

Reply:
ah.... true :)